application/models/Token.php

```diff
@@ -263,6 +263,15 @@ public static function sanitizeToken($token)
         return preg_replace('/[^0-9a-zA-Z_~]/', '', $token);
     }
 
+    /**
+     * Sanitize string for any attribute
+     * @param string $attribute to sanitize
+     * @return string sanitized attribute
+     */
+    public static function sanitizeAttribute($attribute)
+    {
+        return filter_var($attribute, FILTER_SANITIZE_STRING);
```

Comment:
With a checker (is valid) `return filter_var($attribute, FILTER_SANITIZE_STRING) == $attribute` maybe bettre : don't update, just send a error and disable save.
But : unsure admin GUI was ready, same with import …

Reply:
Check : admin GUI not ready …